core_types/Cargo.toml

```diff
@@ -50,3 +50,6 @@ holochain_logging = { version="=0.0.1", path = "../logging" }
 test_utils = { path = "../test_utils"}
 holochain_core = { path = "../core"}
 maplit = "=1.0.1"
+
+[features]
+deadlock_detection = []
```

Comment:
This was activating parkin-lot‘s detection code, right @maackle? Guess it can go now?

Reply:
I was thinking that this would be helpful for selectively activating our own deadlock detection. But it is really just uselessly dangling right now.